WordPress/gutenberg
# Import Maps: Only emit CDATA wrappers for inline scripts for JavaScript

## What?

I'm backporting this fix for WordPress 6.4. So that sites with WP < 6.5 and Gutenberg, and old classic themes that does not support html5.
https://github.com/WordPress/wordpress-develop/pull/5925/


## Testing Instructions




 - Use a WordPress version < 6.5. 6.4.3 is the highest (bug is already fixed on trunk)
 - Use a classic theme that does not support html5. IE: Twenty Twelve
 - Create a post, add an image, activate "expand on click"
 - It should not work in `trunk`, it should work in this PR.
